README.md

```diff
@@ -6,7 +6,13 @@ Initially created to learn the power of macros.
 Focused to be simple and fast. 
 Inspired by other haxe ECS frameworks, especially [EDGE](https://github.com/fponticelli/edge), [ECX](https://github.com/eliasku/ecx), [ESKIMO](https://github.com/PDeveloper/eskimo) and [Ash-Haxe](https://github.com/nadako/Ash-Haxe)
 
-#### Wip
+### Forks
+deepcake/echo is considered complete, but multiple forks are still under active development.
+
+ * [player-03/echoes](https://github.com/player-03/echoes) focuses on ease of use for medium-scale games. Features include entity templates, customizable timesteps, and system priorities.
+ * [onehundredfeet/hmecs](https://github.com/onehundredfeet/hmecs) focuses on performance at scale. Features include object pooling, customizable storage, and singleton components.
+
+Disclaimer: this list is not an endorsement, and deepcake can't guarantee that these forks are backwards-compatible. More details can be found in each fork's readme.
```

Comment:
Seems unnecessary to add this. But up to you :)

Reply:
Wasn't sure if deepcake would want something like that. They did mention backwards compatibility being a concern...

I'll remove it for now.